.github/workflows/lint.yml

```diff
@@ -49,8 +49,7 @@ jobs:
       - name: Run ktlint
         run: |
           cd android
-          ./gradlew lintKotlin --continue
-          ./gradlew :build-logic:convention:lint --continue
+          ./gradlew lint lintKotlin :build-logic:convention:lint --continue
```

Comment:
Combined both to reduce gradle load time. Also added `lint` check on library source.